### DISCUSSION TITLE
Country flag emoji issue

### DISCUSSION THREAD

**Discussion by ibragimof** [adityatelange/hugo-PaperMod#479] [Q&A]
When I wrote something like this: `languageName: ":gb:"`
it displays the **string** without formatting code into emoji (displays as "_:gb:_").
How to get flag emoji instead string?

**Comment by pixelfabs** [comment]
Depends on the operating system. What you see is correct if you're on Windows. Emojis are not system universal. Not all systems have all the emojis. If you like to see a list of emojis supported by Windows, hit Windows key + period sign `Win Key + .`

You will see there are no flags supported. But, I wouldn't worry much about it, most websites are visited on Android & iOS devices.
